.yarnrc.yml

```diff
@@ -1,4 +1,5 @@
 enableGlobalCache: true
 nodeLinker: node-modules
 
+yarnPath: .yarn/releases/yarn-3.4.1.cjs
```

Comment:
It could be removed as it's reading from the package.json

Reply:
And we can also remove the file from .yarn/releases folder

Reply:
I will create a new Pr addressing this.